examples/aws-resource-lab-chart/templates/cluster.yaml

```diff
@@ -101,3 +101,6 @@ spec:
       instancetype: "{{.Values.aws.instanceTypeWorker}}"
     - imageid: "{{.Values.aws.ami}}"
       instancetype: "{{.Values.aws.instanceTypeWorker}}"
+
```

Comment:
No need to change this here, as we are moving to https://github.com/giantswarm/aws-resource-lab to enable guest cluster creation CI, I'll add these changes there.

Reply:
Ack. Thanks for adding to the new repo!